x-pack/plugins/security_solution/public/helpers.tsx

```diff
@@ -221,6 +221,23 @@ const alertFieldMappings: Record<string, string> = {
   'signal.rule.query': `${ALERT_RULE_PARAMETERS}.query`,
 };
 
+/*
+ * Deprecation notice: This functionality should be removed when support for signal.* is no longer
```

Comment:
Would it make sense to add `@deprecated` now already? This function is not deprecated yet, but it might help us remember that we will have to remove it at some point?

Reply:
I would hold off on it for now I think, not to confuse any team members and to remain consistent with the other work that's been done. Thanks for the review!